ingestion/src/metadata/ingestion/source/database/bigquery/metadata.py

```diff
@@ -107,7 +123,9 @@ def get_columns(bq_schema):
             "precision": field.precision,
             "scale": field.scale,
             "max_length": field.max_length,
-            "system_data_type": str(col_type),
+            "system_data_type": _array_sys_data_type_repr(col_type)
```

Comment:
In BQ, `str(col_type)` for a column of type `ARRAY<STRUCT<STRING>>` will be shown as `ARRAY`. Resulting in `def _parse_datatype_string` to not consider the column as an array when checking (line 288) the following

```python
if data_type.startswith("array<"):
```